docker-compose.yml

```diff
@@ -14,53 +67,13 @@ services:
       - POSTGRES_DB=minion
     volumes:
       - pgdata:/var/lib/postgresql/data
-  backend:
-    image: ghcr.io/openfoodfacts/openfoodfacts-server/backend:${TAG}
-    environment:
-      - PRODUCERS_PLATFORM
-      - PRODUCT_OPENER_DOMAIN
-      - PRODUCT_OPENER_PORT
-      - PRODUCT_OPENER_FLAVOR
-      - PRODUCT_OPENER_FLAVOR_SHORT
-      - MONGODB_HOST
-      - POSTGRES_USER
-      - POSTGRES_PASSWORD
-      - ROBOTOFF_URL
-      - GOOGLE_CLOUD_VISION_API_KEY
-      - CROWDIN_PROJECT_IDENTIFIER
-      - CROWDIN_PROJECT_KEY
-      - GEOLITE2_PATH
-    depends_on:
-      - memcached
-      - postgres
-    volumes:
-      - html_data:/opt/product-opener/html/data # Static data (e.g dumps)
-
-      # Binds from frontend container (read-only)
-      - icons_dist:/opt/product-opener/html/images/icons/dist:ro
-      - js_dist:/opt/product-opener/html/js/dist:ro
-      - css_dist:/opt/product-opener/html/css/dist:ro
-      - node_modules:/opt/product-opener/node_modules:ro
-
-      # Users, products, product images and orgs files
-      - users:/mnt/podata/users # .sto
-      - products:/mnt/podata/products # .sto
-      - product_images:/mnt/podata/product_images # .jpg
-      - orgs:/mnt/podata/orgs # .sto
-
-      # Logs
-      - ./logs/apache2:/var/log/apache2
-      - ./logs/apache2:/mnt/podata/logs
-      - ./logs/apache2:/var/log/httpd
-
-      # Apache conf
-      - ./conf/apache.conf:/etc/apache2/sites-enabled/product-opener.conf
-
-      # Tmpfs
-      - type: tmpfs
-        target: /mnt/podata/mnt
-    networks:
-      - webnet
+  backend: *backend-conf
+  incron:
+    <<: *backend-conf
+    # This service run the incron jobs
+    # Only root can run incron
+    user: root
+    command: ["incrond"]
```

Comment:
:'( this means we run another container with the same configuration as product opener ? PO takes in average 6GB of memory so is this really necessary just to run the `incron` command ?

Reply:
The container does not take 6GB, it's the apache/perl processes that does.
Here you just create a single process using incron and running the script, which takes the same size as when it was exec.

You gain the "restart" capability of your service. (using exec to run a service is not a best practice)